ledger/src/blockstore_processor.rs

```diff
@@ -242,6 +242,23 @@ fn process_entries_with_callback(
             entry.transactions.shuffle(&mut rng);
         }
     }
+    let mut pubkeys = HashSet::new();
+    for entry in entries.iter() {
+        for transaction in &entry.transactions {
+            for key in &transaction.message.account_keys {
+                pubkeys.insert(*key);
+            }
+        }
+    }
+    let bank_ = bank.clone();
+    std::thread::Builder::new()
```

Comment:
I don't know the cost of creating a thread like this vs having one always running that I send work to.
This was easy to do for now so I could get some measurements of effectiveness.

Reply:
Rayon spawn might be an option:
https://docs.rs/rayon/1.5.0/rayon/fn.spawn.html

It can create an existing work to be executed in the already-created default threadpool.

Reply:
I think keeping a separate thread pool for I/O workers might be better. Especially since I/O can block on page fault.